lib/rmt/cli/repos.rb

```diff
@@ -43,6 +43,7 @@ def list_repositories(scope: :enabled)
         mirroring_enabled: 'Mirror?',
         last_mirrored_at: 'Last mirrored'
       })
+      puts 'Only enabled repositories are shown by default. Use the `--all` option to see all repositories.' unless options.all
```

Comment:
When you rebase, you'll need to add the bit about csv: `unless options.all || options.csv`.
As well as a test for it :sweat_smile: 
Otherwise LGTM